packages/frontend/src/components/Explorer/ResultsCard/QuickCalculations.tsx

```diff
@@ -90,13 +93,36 @@ const getSqlForQuickCalculation = (
     quickCalculation: QuickCalculation,
     fieldReference: string,
     sorts: SortField[],
+    warehouseType: WarehouseTypes | undefined,
 ) => {
+    let fieldQuoteChar = '"';
+    if (warehouseType) {
+        switch (warehouseType) {
+            case WarehouseTypes.BIGQUERY:
+            case WarehouseTypes.DATABRICKS:
+                fieldQuoteChar = '`';
+                break;
+            case WarehouseTypes.SNOWFLAKE:
+            case WarehouseTypes.REDSHIFT:
+            case WarehouseTypes.POSTGRES:
+            case WarehouseTypes.TRINO:
+                break;
```

Comment:
personal opinion: We should return `"` even if this already set on the `let` variable, for readability.

Reply:
sure!

Reply:
done! https://github.com/lightdash/lightdash/pull/9704/commits/96e540daad1a3e3e633ae5c96684b171e5d6a85d